app/app/graph.py

```diff
@@ -11,30 +11,34 @@ def __init__(self, nodes, edges):
 
     @classmethod
     def load_default(cls):
-        nodes = pd.read_pickle("./resources/nodes.p")
-        edges = pd.read_pickle("./resources/edges.p")
+        nodes = pd.read_pickle("../../resources/nodes.p")
+        edges = pd.read_pickle("../../resources/edges.p")
```

Comment:
We need to find a solution for this. I normally call everything from the project root which makes this fail. On the other hand, I guess you call it from it's directory.